src/me/eglp/gv2/commands/fun/CommandReminder.java

```diff
@@ -0,0 +1,191 @@
+					DefaultMessage.ERROR_INVALID_TIMESTAMP.reply(event);
+					e.printStackTrace();
+					return;
+				}
+				
+				if(dateMs.toEpochSecond(event.getGuild().getConfig().getTimezone().getRules().getOffset(Instant.now())) <= System.currentTimeMillis() / 1000) {
+					//TODO reminder is in the past
+					//event.reply("youz stink");
+					DefaultMessage.COMMAND_REMINDER_CREATE_EVENT_IS_IN_THE_PAST.reply(event);
+					return;
+				}
+				
+				
+				//if(repeat != null) {
+				//	try {
+				//		LocalDateTime repeatMs = LocalDateTime.parse(repeat, DateTimeFormatter.ISO_OFFSET_DATE);
+				//		//repeatMs.toInstant( event.getGuild().getConfig().getTimezone().))
+				//		repeatMs.toInstant(ZoneOffset.UTC);
+				//	}catch(DateTimeParseException e) {
+				//		DefaultMessage.ERROR_INVALID_TIMESTAMP.reply(event);
+				//	}
+				//}
+				
+				GuildReminder reminder = new GuildReminder(event.getGuild(), dateMs, reminderMessage, repeatE, channel);
+				
+				reminder.enqueue();
+				event.getGuild().getRemindersConfig().saveReminder(reminder);
+				
+				DefaultMessage.COMMAND_REMINDER_CREATE_SUCCESS.reply(event);//"Your reminder got succesefully enqueued!");
+				
+				//event.deleteMessage(JDAEmote.OK_HAND);
+			}
+			
+			@Override
+			public List<OptionData> getOptions() {
+				OptionData optionRepeat = new OptionData(OptionType.STRING, "repeat", "Whether and when to repeat the reminder", false);
+				for(ReminderRepetitionEnum a : ReminderRepetitionEnum.values()) {
+					optionRepeat.addChoice(a.toString(), a.name());
+				}
+				List<OptionData> ops = new ArrayList<>(Arrays.asList(
+						new OptionData(OptionType.STRING, "date_and_time", "When the reminder is triggered", true),
+						new OptionData(OptionType.STRING, "reminder_message", "The message of the reminder", true),
+						optionRepeat,
+						new OptionData(OptionType.CHANNEL, "channel", "The channel in which to send the reminder", false).setChannelTypes(ChannelType.TEXT, ChannelType.NEWS, ChannelType.GUILD_NEWS_THREAD, ChannelType.GUILD_PRIVATE_THREAD, ChannelType.GUILD_PUBLIC_THREAD)));
+				return ops;
+			}
+		})
+		.setDescription(DefaultLocaleString.COMMAND_REMINDER_CREATE_DESCRIPTION)
+		.setUsage(DefaultLocaleString.COMMAND_REMINDER_CREATE_USAGE)
+		.setPermission("fun.reminder.create");
+		
+		addSubCommand(new Command(this, "list") {
+			
+			@Override
+			public void action(CommandInvokedEvent event) {
+				List<GuildReminder> reminders = event.getGuild().getRemindersConfig().getReminders();
+				if(reminders.isEmpty()) {
+					DefaultMessage.COMMAND_REMINDER_LIST_NO_REMINDERS.reply(event);
+					return;
+				}
+				
+				EmbedBuilder theEmbed = new EmbedBuilder().setColor(Color.PINK).setDescription(DefaultLocaleString.COMMAND_REMINDER_LIST_EMBED_DESCRIPTION.getFor(event.getGuild())).setTitle(DefaultLocaleString.COMMAND_REMINDER_LIST_EMBED_TITLE.getFor(event.getGuild()));
+				for(GuildReminder currentReminder : reminders) {
+					theEmbed.addField(currentReminder.getId(),
```

Comment:
I think using a list (such as in the `poll` command) is better :man_shrugging:

Reply:
A Generic List is very confusing for the users, since this feature uses two "keys" (id and channel) and not just one (id, like the poll command does), to make it more clear, which id is used for what reminder

Reply:
It should still be changed due to embed limits (max of 25 fields per embed)